lib/engine.py

```diff
@@ -109,15 +109,15 @@ def source_gen():
                 self.table.record_id += 1
 
                 # Check for single row distributed over multiple lines
-                val_list = self.table.split_on_delimiter(line)
+                val_list = self.table.extract_values(line)
                 while len(val_list) < len(self.table.get_column_datatypes()):
                     line = line.rstrip('\n')
                     if type(real_lines) != 'list':
```

Comment:
If I understand correctly, you're pointing out that the `'list'` vs `list` issue from #579 isn't fixed here. That's just because I started working on this before #579 was merged and this is forked off of master. The merge should leave the changes from #579 in tact.